api/src/main/java/org/pmiops/workbench/actionaudit/bucket/BucketAuditEntry.java

```diff
@@ -0,0 +1,69 @@
+package org.pmiops.workbench.actionaudit.bucket;
+
+import java.time.OffsetDateTime;
+import java.time.temporal.ChronoUnit;
+
+public class BucketAuditEntry {
+
+  private String petAccount;
+  private long fileLengths;
+  private String googleProjectId;
+  private String bucketName;
+  private OffsetDateTime minTime;
+  private OffsetDateTime maxTime;
+
+  public String getPetAccount() {
+    return petAccount;
+  }
+
+  public void setPetAccount(String petAccount) {
+    this.petAccount = petAccount;
+  }
+
+  public long getFileLengths() {
+    return fileLengths;
+  }
+
+  public void setFileLengths(long fileLengths) {
+    this.fileLengths = fileLengths;
+  }
+
+  public OffsetDateTime getMinTime() {
+    return minTime;
+  }
+
+  public void setMinTime(OffsetDateTime minTime) {
+    this.minTime = minTime;
+  }
+
+  public OffsetDateTime getMaxTime() {
+    return maxTime;
+  }
+
+  public void setMaxTime(OffsetDateTime maxTime) {
+    this.maxTime = maxTime;
+  }
+
+  public String getGoogleProjectId() {
+    return googleProjectId;
+  }
+
+  public void setGoogleProjectId(String googleProjectId) {
+    this.googleProjectId = googleProjectId;
+  }
+
+  public String getBucketName() {
+    return bucketName;
+  }
+
+  public void setBucketName(String bucketName) {
+    this.bucketName = bucketName;
+  }
+
+  public long getTimeWindowDuration() {
```

Comment:
Can we use Duration, since it is Duration?

Reply:
we can, but I don't think it's worth the conversion, because we need this time seconds for the egress event creation in ObjectNameLengthService.maybePersistEgressEvent